src/openrct2-ui/WindowManager.cpp

```diff
@@ -837,6 +837,8 @@ class WindowManager final : public IWindowManager
         std::unique_ptr<WindowBase>&& wp, WindowClass cls, ScreenCoordsXY pos, int32_t width, int32_t height,
         uint32_t flags) override
     {
+        // auto titleBarHeight = (flags & WF_NO_TITLE_BAR) ? 0 : GetTitleBarHeight();
```

Comment:
I've deferred this to `ResizeFrame`. In the end, I didn't use the flag either. We can discuss whether or not to keep it.

Reply:
Not a fan of removing the flag. Ultimately, I think a lot of window management will be more centralised and then the code will need a reliable way to determine whether to call `ResizeFrame()` or not. Or we might even have a situation where the window frame is not even created by the window subclasses, but also centrally.

Reply:
I'm fine with keeping the flags around. Either way, the comment can be removed at this point.